.sql

```diff
@@ -0,0 +1,19 @@
+createdb twoiam 
+psql twoiam 
+
+drop table if exists users;
+drop table if exists images;
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  github_name VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE images (
+  id SERIAL PRIMARY KEY,
+  users_id INT REFERENCES users(id),
```

Comment:
This should be singular user_id because it will only ever contain a single user I'd